resources/views/script.blade.php

```diff
@@ -1,43 +1,48 @@
 @if(Astrogoat\Elevar\Settings\ElevarSettings::isEnabled())
```

Comment:
When you have a lot of setting calls, it can sometimes be advantageous to define it as a variable first.
Also, you can combine the two "if" statements into one.

```diff
- @if(Astrogoat\Elevar\Settings\ElevarSettings::isEnabled())
- 
- <script id="elevar-dl-listener-config" type="application/json">{"data_layer_listener_script": "https://shopify-gtm-suite.getelevar.com/shops/{{ settings(Astrogoat\Elevar\Settings\ElevarSettings::class, 'uuid') }}/events.js", "ss_url": null}</script>

+ @php($settings = settings(Astrogoat\Elevar\Settings\ElevarSettings::class))
+
+ @if($settings->isEnabled() && $settings->data_layer_listener_enabled)
+     <!-- [ELEVAR] Start -->
+     <script id="elevar-dl-listener-config" type="application/json">
+         {
+             "data_layer_listener_script": "https://shopify-gtm-suite.getelevar.com/shops/{{ $settings->uuid }}/events.js",
+             "ss_url": {!! blank($settings->server_side_url) ? 'null' : '"' . $settings->server_side_url . '"' !!}
+         }
+     </script>
```

Reply:
thank You 😄